pkg/kubernetes/scripts/tags.js

```diff
@@ -64,7 +64,7 @@
         return names;
     }
 
-    function buildSpec(names, spec) {
+    function buildSpec(names, spec, insecure) {
```

Comment:
Can we make this so if insecure is null or undefined the relevant importPolicy is not set or reset?